docs/esignet-openapi.yaml

```diff
@@ -4417,6 +4417,107 @@ paths:
+  /binding/v2/binding-otp:
+    post:
+      tags:
+        - WALLET BACKEND
+      summary: Send Binding OTP Endpoint
+      description: Send wallet binding OTP endpoint is invoked by Mimoto server.
+      operationId: post-binding-otp
+      parameters:
+        - name: partner-api-key
+          in: header
+          description: 'API key of the binding partner, this will be passed to binder implementation to interact with authentication system.'
+          schema:
+            type: string
+        - name: partner-id
+          in: header
+          description: 'Binding partner Identifier, this will be passed to binder implementation to interact with authentication system.'
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                requestTime:
+                  type: string
+                request:
+                  type: object
+                  properties:
+                    individualId:
+                      type: string
+                      description: User Id (UIN/VID)
+                    otpChannels:
+                      type: array
+                      description: Channels to which OTP should be delivered.
+                      items:
+                        type: string
+                  required:
+                    - individualId
+                    - otpChannels
+              required:
+                - requestTime
+                - request
+            examples:
+              Example 1:
+                value:
+                  requestTime: '2023-09-22T08:01:13.000Z'
+                  request:
+                    individualId: '24554655645'
+                    otpChannels:
+                      - sms
+                      - email
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  responseTIme:
+                    type: string
+                  response:
+                    type: object
+                    properties:
+                      maskedEmail:
+                        type: string
+                        description: Masked email id of the individualId user.
+                      maskedMobile:
+                        type: string
+                        description: Masked mobile number of the individualId user.
+                  errors:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        errorCode:
+                          type: string
+                          enum:
+                            - invalid_otp_channel
+                            - unknown_error
+                            - invalid_individual_id
+                            - send_otp_failed
```

Comment:
Captcha error codes are missing @Md-Humair-KK

Reply:
Added the errorcodes. I have copied them but forgot to add additional required captcha related details.